spec/models/review_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  describe 'create' do
```

Comment:
You only need to have `describe 'create'` once,
and then have multiple `context` expressions inside it